test/parallel/test-net-connect-handle-econnrefused.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
-var common = require('../common');
-var net = require('net');
-var assert = require('assert');
+const common = require('../common');
+const net = require('net');
+const assert = require('assert');
 
 
 // Hopefully nothing is running on common.PORT
-var c = net.createConnection(common.PORT);
+const c = net.createConnection(common.PORT);
 
 c.on('connect', function() {
   console.error('connected?!');
-  assert.ok(false);
+  common.fail();
```

Comment:
Could you pass a message to `common.fail()`. You can also remove the `console.error()` call on the previous line.